CameraButtonsViewControllerTests.swift

```diff
@@ -0,0 +1,282 @@
+        XCTAssertTrue(innerLeftVerticalStackview?.arrangedSubviews
+            .contains(cameraButtonsViewController.fileImportButtonView) ?? false,
+                      "the inner stck view should contain the file import view")
+    }
+    
+    func testLayoutWhenFileImportAndFlashAreEnabledOnIpad() {
+        let giniConfiguration = GiniConfiguration()
+        giniConfiguration.fileImportSupportedTypes = .pdf_and_images
+        giniConfiguration.flashToggleEnabled = true
+        cameraButtonsViewController = CameraButtonsViewController(giniConfiguration: giniConfiguration,
+                                                                  isFlashSupported: true,
+                                                                  currentDevice: IpadDevice())
+        _ = cameraButtonsViewController.view
+        cameraButtonsViewController.addFileImportButton()
+        
+        let innerRightVerticalStackview = cameraButtonsViewController.rightStackView
+            .arrangedSubviews.first as? UIStackView
+        let innerLeftVerticalStackview = cameraButtonsViewController.leftStackView
+            .arrangedSubviews.first as? UIStackView
+        
+        XCTAssertNotNil(innerRightVerticalStackview, "right stack view should not contain an inner stack view")
+        XCTAssertNotNil(innerLeftVerticalStackview, "left stack view should contain an inner stack view")
+        XCTAssertEqual(innerRightVerticalStackview?.arrangedSubviews.count, 1,
+                       "the inner stack view should contain only the flash toggle button")
+        XCTAssertEqual(innerRightVerticalStackview?.arrangedSubviews[0],
+                      cameraButtonsViewController.flashToggleButtonContainerView,
+                      "the inner stack view should contain the flash toggle button")
+        XCTAssertEqual(innerLeftVerticalStackview?.arrangedSubviews[0],
+                       cameraButtonsViewController.fileImportButtonView,
+                       "the inner stack view should contain the file import view")
+    }
+    
+    func testLayoutWhenFileImportMultipageAndFlashAreEnabledOnIpad() {
+        let giniConfiguration = GiniConfiguration()
+        giniConfiguration.fileImportSupportedTypes = .pdf_and_images
+        giniConfiguration.flashToggleEnabled = true
+        giniConfiguration.multipageEnabled = true
+        cameraButtonsViewController = CameraButtonsViewController(giniConfiguration: giniConfiguration,
+                                                                  isFlashSupported: true,
+                                                                  currentDevice: IpadDevice())
+        _ = cameraButtonsViewController.view
+        cameraButtonsViewController.addFileImportButton()
+        
+        let innerRightVerticalStackview = cameraButtonsViewController.rightStackView
+            .arrangedSubviews.first as? UIStackView
+        let innerLeftVerticalStackview = cameraButtonsViewController.leftStackView
```

Comment:
This and the previous two methods duplicate these lines of code. Maybe you can extract it as a method?